ui/app/components/token-editor.hbs

```diff
@@ -0,0 +1,236 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form class="acl-form" autocomplete="off" {{on "submit" this.save}}>
+	<label>
+    <Hds::Form::Label>
+      Token Name
+    </Hds::Form::Label>
+		<Input
+			data-test-token-name-input
+			@type="text"
+			@value={{@token.name}}
+			class="input"
+		/>
+	</label>
+
+{{!-- TODO: add expiration time here --}}
```

Comment:
Isnt it already there? at least time until expiration

Reply:
Yep good catch — this was an earlier TODO